apps/api/test/fixture/reports.ts

```diff
@@ -26,3 +28,61 @@ export const addMilestone = {
   notes: 'Test Note 1',
   reason: '',
 };
+
+// Report 4 data
+
+export const statusNames = [
+  'BCCNM Full Licence LPN',
+  'BCCNM Full Licence RN',
+  'BCCNM Provisional Licence LPN',
+  'BCCNM Provisional Licence RN',
+];
+export const jobInput: IENApplicantJobCreateUpdateDTO = {
+  ha_pcn: '6ad69443-e3a8-3cbc-8cc9-3a05e5b771e4',
+  job_id: '148593',
+  recruiter_name: 'Test 4',
+};
+
+export type ReportTestStatus = LICENSE_STATUS | BCCNM_STATUS;
+
+export enum LICENSE_STATUS {
```

Comment:
With enum and constants in `milestone-status.ts`, I can't catch why we need this additional enum.

Reply:
We don't need it, you are correct. I am now aware of the other enums and will correct this. Thanks